packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx

```diff
@@ -627,13 +627,16 @@ describe.skip('Line chart - Subcomponent Event', () => {
   testWithWait(
     'Should render events with defined data',
     LineChart,
-    { data: simplePoints, eventAnnotationProps, tickValues, tickFormat: '%m/%d' },
+    { data: simplePoints, eventAnnotationProps: eventAnnotationProps },
```

Comment:
what was the issue. How is the test case passing now

Reply:
queryByText is not returning the events data so we have updated it to getElementsByTagName also removed unwanted props.

Reply:
get by tag should not be used for RTL.
See the suggested methods to use https://testing-library.com/docs/react-testing-library/cheatsheet